packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -164,7 +164,7 @@ const renderHeaderContent = ({
             {`${stateLabel.toUpperCase()}`}
```

Comment:
Uppercase does not exist for several scripts, e.g arabic. We should check that it makes sense for all scripts when carrying out a UX review.

Reply:
Looks like when it is a script that does not support capital letters, they are just not getting updated e.g. arabic text just stays the same as it is. But will check for other services too